scripts/build-data.js

```diff
@@ -33,12 +33,12 @@ export default ${serialize(localeData)};
 function writeUMDFile(filename, module) {
   const lang = p.basename(filename, '.js');
 
+  console.log(lang)
```

Comment:
remove?

Reply:
How about at least replacing it with a progress bar? This takes ages...

Reply:
@redonkulus would you like me to replace it with a progress bar? For example https://www.npmjs.com/package/progress

Reply:
@redonkulus should we in addition maybe provide prebuilt data for the `prepare` script in order to save consumers from building the data themselves on `npm install`?

Reply:
What was the reason for adding the `console.log` to begin with? Thats why I asked if it needed to be removed, was this merely to aid in your debugging or something you wanted to add for devs?

Reply:
I added it for everyone.
For some reason the build of the locale data slowed down a lot by upgrading bagel, rollup etc. That's why I added some kind of progress measurement for our users.

However there are alternatives:
1) making it a progress bar
2) avoiding to build the locale data on user machines all together and instead provide them using GitHub releases

I'm willing to implement them; However I need your "go" because I don't want to do it for nothing ;)

Reply:
Done